server/src/server.rs

```diff
@@ -774,7 +775,26 @@ impl CubebServer {
             cubeb::StreamParamsRef::from_ptr(osp as *const StreamParams as *mut _)
         });
 
-        let latency = params.latency_frames;
+        // TODO: Manage stream latency requests with respect to the RT deadlines the callback_thread was configured for.
+        fn round_up_pow2(v: u32) -> u32 {
```

Comment:
Just curious. Why does this need to be power of 2?

Reply:
It's not strictly necessary, just a method of rounding the minimum buffer size up a bit.